app/src/main/java/org/sopt/havit/ui/category/CategoryContentModifyActivity.kt

```diff
@@ -56,10 +57,14 @@ class CategoryContentModifyActivity :
         binding.ivBack.setOnClickListener { finish() }
     }
 
+    private fun deleteText(){
+        binding.ivDeleteText.setOnClickListener { binding.etCategory.text.clear() }
+    }
+
     private fun setTextWatcher() {
         binding.etCategory.addTextChangedListener {
             // 중복된 카테고리 명인지 검사
-            binding.isDuplicated = (binding.etCategory.text.toString() in categoryTitleList)
+            binding.isDuplicated = (binding.categoryTitle in categoryTitleList && binding.categoryTitle != categoryName)
```

Comment:
&& 뒤에 내용은 현재 카테고리 명이 아닌경우를 말하는거죠?
조금 더 직관적인 방법을 제안하자면, 
```
fun exceptCurrentCatetory() {
    binding.categoryTitle != categoryName
}
``` 
함수를 추가하고, 
```suggestion
            binding.isDuplicated = (binding.categoryTitle in categoryTitleList && exceptCurrentCatetory())
``` 
이렇게 메소드로 분리시키는것도 하나의 방법일것 같고
방법이 맘에 들지 않는다면, 주석으로 살짝 적어줘도 좋을것같아요!

Reply:
아하 제 코드 스타일이 적게 쓸 수 있으면 적게 쓰는걸 좋아해서 주석으로 추가해둘게욥~!

Reply:
저두 적게쓰는걸 좋아합니다만, 적당한 가독성과 합의를 봐야할 때가 있는것 같네요😂😂
한참 고민했슴니다,,